### DISCUSSION TITLE
My codes have been stolen

### DISCUSSION THREAD

**Discussion by frqnsa** [community/community#49852] [Code Security] [answered]
### Select Topic Area

Question

### Body

Hello everyone. Someone I don't know has stolen my code from my Github Pages site using the "HTTrack" program and opened a new "repository" without my permission. I want to report this person's Github account and prevent my code from being distributed to more people. Please help me. I have not authorized this person to use or modify my codes. He stole my codes with a program and distributed my private codes to everyone on the Discord platform. Please Github Team contact me :)

![IMG_20230312_172340](https://user-images.githubusercontent.com/108571038/224547573-7d967cb3-c443-4523-a33d-6cdb20931064.jpg)

![IMG_20230312_172355](https://user-images.githubusercontent.com/108571038/224547586-bea07203-6226-420f-a29a-07d5dfa8244a.jpg)

**Comment by bbkx226** [comment, marked as answer]
Hi hakkai34, I think you can first gather evidence (which you've done with these screenshots), then request GitHub to take down the repository containing your stolen code. If you still having the problem, I think you can reach out GitHub Support and ask them the issue. Finally, I'd like to suggest you to protect your code by adding a license to it, which specifies the terms under which others can use, modify, and distribute your code

**Reply by frqnsa** [reply]
As a matter of fact, I tried to reach him, but there was no response. How can I apply a license to prevent my codes from being stolen later?

**Reply by bbkx226** [reply]
On your GitHub repo, you can create a new file, and type license as your file name, a button called "choose your template" or something like that will appear on your screen, just click it and select which one is suitable for you

If you don't know which one is suitable for you, there's a page that explains how each license work https://choosealicense.com/
